docs/topics/pages.rst

```diff
@@ -440,26 +440,25 @@ This is because ``Page`` enforces ordering QuerySets by path. Instead you must a
 
     news_items = NewsItemPage.objects.live().order_by('-publication_date')
 
-Page custom managers
+Custom Page managers
 --------------------
 
 ``Page`` enforces its own 'objects' manager in its ``__init__`` method, so you cannot add a custom manager at the 'objects' attribute.
```

Comment:
> The code for filtering for future events is already in a future() method.

So it is, sorry - I must have gone cross-eyed looking at the diffs :-) Thanks for the clarifications!